tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -39,26 +73,53 @@ mod tests {
     use crate::fmt::format::Writer;
     use crate::fmt::time::FormatTime;
 
+    #[cfg(feature = "chrono")]
+    use super::ChronoFmtType;
     #[cfg(feature = "chrono")]
     use super::ChronoLocal;
     #[cfg(feature = "chrono")]
     use super::ChronoUtc;
 
     #[cfg(feature = "chrono")]
     #[test]
-    fn test_chrono_format_time_utc() {
+    fn test_chrono_format_time_utc_default() {
         let mut buf = String::new();
         let mut dst: Writer<'_> = Writer::new(&mut buf);
-        assert!(FormatTime::format_time(&ChronoUtc, &mut dst).is_ok());
+        assert!(FormatTime::format_time(&ChronoUtc::default(), &mut dst).is_ok());
         // e.g. `buf` contains "2023-08-18T19:05:08.662499+00:00"
     }
 
     #[cfg(feature = "chrono")]
     #[test]
-    fn test_chrono_format_time_local() {
+    fn test_chrono_format_time_utc_custom() {
+        let fmt = ChronoUtc {
```

Comment:
a non-blocking nit, given that I took too long to review this, but maybe rewriting these tests in terms of https://github.com/tokio-rs/tracing/blob/245d6075533651b2ebfaed7f06fc80e6ea76fc53/tracing-subscriber/src/fmt/time/datetime.rs#L341 might be warranted?

Reply:
if you don't mind, i'll leave that for now.